pkg/release/consts.go

```diff
@@ -79,6 +79,23 @@ var (
 	// UBIArchitectures is defined separately as there is no 'arm' variant of
 	// the UBI8 base image.
 	UBIArchitectures = []string{"amd64", "arm64", "ppc64le", "s390x"}
+
+	// ClientPlatforms is the list of OSes and architectures to build client CLI tools
+	// for during the release.
+	// This is used to determine which artifacts should be uploaded.
+	ClientPlatforms = map[string][]string{
+		"linux":   []string{"amd64", "arm", "arm64", "ppc64le", "s390x"},
+		"darwin":  []string{"amd64"},
+		"windows": []string{"amd64"},
+	}
+
+	// ArchitecturesPerOS is the list of OSes and architectures that we can build
+	// This is used to drive the `--platforms` flag passed to 'bazel build'
+	ArchitecturesPerOS = map[string][]string{
+		"linux":   []string{"amd64", "arm", "arm64", "ppc64le", "s390x"},
+		"darwin":  []string{"amd64"},
+		"windows": []string{"amd64"},
```

Comment:
I'm confused what the need for this is when we already have `UBIArchitectures`, `ClientPlatforms` etc. Would it not make more sense for us to have all of the current `*Architectures` fields be `*Platforms` fields (i.e. be an OS->arch mapping), and then have some code that 'flattens' all the different combinations in order to compute the `--platforms` flag?

Reply:
Made all platforms in the same mapped structure

Reply:
:+1: could this var now be replaced by something that automatically derives the overall platform combinations based on `ClientPlatforms`, `ServerPlatforms` etc.? This seems like it could fall out of sync otherwise